src/org/zaproxy/zap/extension/exportreport/ExtensionExportReport.java

```diff
@@ -0,0 +1,290 @@
+	{
+		initializeLists();
+		frameER = new FrameExportReport(this, extensionGetCardSource(),extensionGetCardAlertRisk(),extensionGetCardAlertDetails());
+	}
+
+	private void initializeLists()
+	{
+		// PanelAlertRisk
+		alertSeverity.clear();
+		alertSeverity.add(Constant.messages.getString("exportreport.risk.severity.high.label"));
+		alertSeverity.add(Constant.messages.getString("exportreport.risk.severity.medium.label"));
+		alertSeverity.add(Constant.messages.getString("exportreport.risk.severity.low.label"));
+		alertSeverity.add(Constant.messages.getString("exportreport.risk.severity.info.label"));
+
+		// PanelAlertDetails
+		alertDetails.clear();
+		alertDetails.add(Constant.messages.getString("exportreport.details.cweid.label"));
+		alertDetails.add(Constant.messages.getString("exportreport.details.wascid.label"));
+		alertDetails.add(Constant.messages.getString("exportreport.details.description.label"));
+		alertDetails.add(Constant.messages.getString("exportreport.details.otherinfo.label"));
+		alertDetails.add(Constant.messages.getString("exportreport.details.solution.label"));
+		alertDetails.add(Constant.messages.getString("exportreport.details.reference.label"));
+
+		alertAdditional.clear();
+		alertAdditional.add(Constant.messages.getString("exportreport.details.requestheader.label"));
+		alertAdditional.add(Constant.messages.getString("exportreport.details.responseheader.label"));
+		alertAdditional.add(Constant.messages.getString("exportreport.details.requestbody.label"));
+		alertAdditional.add(Constant.messages.getString("exportreport.details.responsebody.label"));
+
+		maxList = (alertDetails.size() + alertAdditional.size());
+	}
+	
+	private PanelSource extensionGetCardSource()
+	{
+		cardSource = new PanelSource(this);
+		return cardSource;
+	}
+
+	private PanelAlertRisk extensionGetCardAlertRisk()
+	{
+		cardAlertRisk = new PanelAlertRisk(this, alertSeverity);
+		return cardAlertRisk;
+	}
+
+	private PanelAlertDetails extensionGetCardAlertDetails()
+	{
+		cardAlertDetails = new PanelAlertDetails(this, alertDetails, alertAdditional);
+		return cardAlertDetails;
+	}
+
+	public int extensionGetMaxList()
+	{
+		return maxList;
+	}
+
+	/* Source Card return data */
+	public String extensionGetTitle()
+	{
+		return cardSource.getTitle();
+	}
+
+	public String extensionGetBy()
+	{
+		return cardSource.getBy();
+	}
+
+	public String extensionGetFor()
+	{
+		return cardSource.getFor();
+	}
+
+	public String extensionGetScanDate()
+	{
+		return cardSource.getScanDate();
+	}
+
+	public String extensionGetReportDate()
+	{
+		return cardSource.getReportDate();
+	}
+
+	public String extensionGetScanVer()
+	{
+		return cardSource.getScanVer();
+	}
+
+	public String extensionGetReportVer()
+	{
+		return cardSource.getReportVer();
+	}
+
+	public String extensionGetDescription()
+	{
+		return cardSource.getDescription();
+	}
+
+	/* Alert Risk Card return data */
+	public ArrayList<String> getIncludedAlertSeverity()
```

Comment:
Yes, because it's a dual list modal and this was the recommended way of calling it from the java tutorial. Also I tried calling the individual lists and there were problems, i can go back and debug this down the line to improve for 2.0.